cmake/std/atdm/mutrino/environment.sh

```diff
@@ -55,3 +56,103 @@ export ATDM_CONFIG_NETCDF_LIBS="-L${BOOST_ROOT}/lib;-L${NETCDF_ROOT}/lib;-L${PNE
 
 export ATDM_CONFIG_MPI_POST_FLAG="-c 4"
 export ATDM_CONFIG_COMPLETED_ENV_SETUP=TRUE
+
+#
+# Run a script on the compute node send STDOUT and STDERR output to a file
+# while also echo output to the console.  The primary purpose is to run the
+# tests on the compute node.
+#
+# Usage:
+#
+#   atdm_run_script_on_comput_node <script_to_run> <output_file> \
+#     [<timeout>] [<account>]
+#
+# If <timeout> and/or <account> are not given, then defaults are provided that
+# work for the Jenkins driver process.
+#
+# In this case, sbatch is used to run the script but it also sends ouptut to
+# STDOUT in real-time while it is running in addition to writing to the
+# <outout_file>.  The job name for the sbatch script is taken from the env var
+# 'JOB_NAME'.  This works for local builds since JOB_NAME.
+#
+# Note that you can pass in the script to run with arguments such as with
+# "<some-script> <arg1> <arg2>" and it will work.  But note that this has to
+# be bash script that 'sbatch' can copy and run form a temp location and it
+# still has to work.  So the script has to use absolute directory paths, not
+# relative paths or asume sym links, etc.
+#
+function atdm_run_script_on_compute_node {
```

Comment:
@fryeguy52, this function `atdm_run_script_on_compute_node` looks identical char-for-char identical to the one in `cmake/std/atdm/toss3/environment.sh`.  Is that correct?  Was it a complete 100% copy and paste?  If that is the case, we should factor this function out into its own file and then include it in `cmake/std/atdm/toss3/environment.sh` and `cmake/std/atdm/mutrino/environment.sh` in a follow-up refactoring and PR (I don't want to delay getting this PR merged so that we can see this running tomorrow).